src/Types/Object_.php

```diff
@@ -55,15 +54,10 @@ public function getFqsen()
         return $this->fqsen;
     }
 
-    /**
-     * Returns a rendered output of the Type as it would be used in a DocBlock.
```

Comment:
Think the docblock here makes sense. The `@return`  doesn't

Reply:
What I actually did here was to *move* the docblock to the interface method, rather than it being located in this implementor.